slue_toolkit/generic_utils.py

```diff
@@ -4,6 +4,59 @@
 
 import pickle as pkl
 
+raw_entity_to_spl_char = {
```

Comment:
@ankitapasad This might be more easy to recognize the mapping than .pkl file. What do you think?

Reply:
I agree that this is a much more readable format. My only concern is that explicitly writing it out makes it prone to silly errors in case any line gets edited by the user. Our evaluation uses this same mapping so it is imperative that the user uses the same map throughout and so we keep these maps as less prone to erroneous (or purposeful) edits as possible. That was my motivation for using pkl files, since those are more shielded from such possible mistakes.

@fwu-asapp I don't quite follow the second concern "the pkl files can't be loaded correctly, if people import slue_toolkit outside (not from the root of this repo)": Could you please elaborate? Thanks!

If you feel strongly about this (over pkl) then I am fine too, just take the above point into consideration that's all.

Reply:
Got it, thanks for the clarification!